app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title page_title = ""
-    base_title = t".base_title"
+    base_title = t ".base_title"
     
     page_title.empty? ? base_title : page_title + " | " + base_title
```

Comment:
đoạn này sửa thành nội suy đi ô (:v:)
```suggestion
     page_title.empty? ? base_title : "#{page_title} | #{base_title}" 
```